src/PDFViewer.ts

```diff
@@ -118,6 +155,10 @@ class PDFViewer {
     } catch (error) {
       throw new Error(`PDFViewer error: PDF document can not be loaded - ${error}`);
     }
+
+    setTimeout(() => {
```

Comment:
What about instead of setTimeout we use:
`pdfJsApp.on('documentloaded', () => pdfJsApp.setInitialScale(this.options.initialScale));`